pkgs/cronet_http/example/integration_test/cronet_configuration_test.dart

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Tests various [CronetEngine] configurations.
+
+import 'dart:io';
+
+import 'package:cronet_http/cronet_client.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:test/test.dart';
+
+void testCache() {
+  group('cache', () {
+    late HttpServer server;
+    var numRequests = 0;
+
+    setUp(() async {
+      numRequests = 0;
+      server = (await HttpServer.bind('localhost', 0))
+        ..listen((request) async {
+          await request.drain<void>();
+          ++numRequests;
+          request.response.headers.set('Content-Type', 'text/plain');
+          request.response.headers
+              .set('Cache-Control', 'public, max-age=30, immutable');
+          request.response.headers.set('etag', '12345');
+          await request.response.close();
+        });
+    });
+    tearDown(() {
+      server.close();
+    });
+
+    test('disabled', () async {
+      final builder = await CronetEngine.build(cacheMode: CacheMode.disabled);
```

Comment:
I think these are engines, not builders (here and below).

Reply:
Done